README.md

```diff
@@ -138,7 +138,7 @@ For the most customized way of installing RAPIDS and cuxfilter, visit the select
 ```bash
 # ex. for CUDA 10.2
 conda install -c rapidsai -c nvidia -c conda-forge \
-    -c defaults cuxfilter=0.15 python=3.6 cudatoolkit=10.2
+    -c defaults cuxfilter=0.18 python=3.6 cudatoolkit=10.2
```

Comment:
if you add an entry to [update-version.sh](https://github.com/rapidsai/cuxfilter/blob/branch-0.18/ci/release/update-version.sh#L49-L56) for this line, it will be automatically updated during each release.

Reply:
@ajschmidt8 thanks! updated the `update-version.sh`, can you have a quick look to see if everything looks good?